Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -45,6 +45,187 @@ COREGRAPHICS_EXPORT void CGImageAddDestructionListener(CGImageDestructionListene
 
 #pragma endregion OLD_CODE
 
+#pragma region CGImageImplementation
+
+struct __CGImageImpl {
```

Comment:
WIP, I though of just pushing this through for now and send that out later. Given I don't think i'll have that out  till Monday.

---
In reply to: [88774084](https://github.com/Microsoft/WinObjC/pull/1430#discussion_r88774084) [](ancestors = 88774084)